test/inductor/test_compiled_optimizers.py

```diff
@@ -11,6 +11,7 @@
 import torch
 
 import torch._inductor
+import torch._inductor.cudagraph_trees
```

Comment:
not intended ?

Reply:
this actually is intended, sometimes the import is missing?

Reply:
Looks like lines 13-15 are mergeable into one import